src/Symfony/Component/Mailer/Bridge/Amazon/Transport/SesApiTransport.php

```diff
@@ -113,6 +113,9 @@ private function getPayload(Email $email, Envelope $envelope): array
         if ($email->getHtmlBody()) {
             $payload['Message.Body.Html.Data'] = $email->getHtmlBody();
         }
+        if (!empty($email->getReplyTo())) {
```

Comment:
Should be `if ($email->getReplyTo()) {` to be consistent with the other checks above.